src/doomkeys.h

```diff
@@ -24,70 +24,70 @@
 // This is the stuff configured by Setup.Exe.
 // Most key data are simple ascii (uppercased).
 //
-#define KEY_RIGHTARROW  0xae
```

Comment:
Now that we have clear text key names in the config file, we do not have to adhere to the original DOS scan codes anymore (i.e. we don't store their numerical values anyway). If we did, we'd end up with ambiguities e.g. with the keypad keys or other keys which send multi-byte scan codes.